src/config/dynamic-theme-fixes.config

```diff
@@ -17337,6 +17337,36 @@ INVERT
 .widget-ihsg-chart img
 .konvajs-content canvas
 .flotcompanychart canvas
+.ant-modal-content img
+.analyst-title img
+.card-calendar-title img
+.calendar-more span
+.save-orderbook img
+.company-navigation-more span
+.dropmenu-more span
+.navmore span
+.stream-more-wrap span
+.stream-more span
+.trading-menu-img
+.securities-input-pass-wrapper img
+.styled__PasswordContainer-w16pvr-2 img
+.connectwrap_slim-off
+.searchstream
+.ssnlogo
+
+CSS
+html {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+body {
+    background-color: var(--darkreader-neutral-background) !important;
+}
```

Comment:
Are you sure about this? Darkreader should already do this by default.

Reply:
unfortunately that's a side-effect of Dark Reader. I personally prefer to not allow such fix, as otherwise every other domain that has this problem will have the same fix, while it actually should be fixed within the code instead of custom fixes.